recipes/ipydatetime/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "ipydatetime" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b083e08efef140c29091fe114ea2921f88989e56f0b5eecdde9ac0157bb0b10
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - jupyter-packaging
+    - nodejs
```

Comment:
does not need `nodejs` in (at least) `host`

Reply:
`nodejs` should only be in `build`.

Reply:
Do you mean I should remove it from both `host` and `run`?

Reply:
yes

Reply:
Ok, let me try that